lib/apiservers/service/restapi/handlers/vch_get.go

```diff
@@ -0,0 +1,428 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"bytes"
+	"context"
+	"encoding/pem"
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+
+	"github.com/docker/docker/opts"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/go-openapi/strfmt"
+	"github.com/vmware/govmomi/vim25/types"
```

Comment:
`gofmt` will complain that there should be a space above this line separating vmware imports from external imports.